### PR TITLE
SearXNG theme oscar

### PR DESCRIPTION
**Pull request by dalf**
## What does this PR do?

SearXNG brand: Oscar theme

Related to https://github.com/searxng/searxng/pull/358

## Why is this change important?

See https://github.com/searxng/searxng/pull/358

### CODE REVIEW

File: searx/static/themes/oscar/src/js/01_init.js [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Affero General Public License for more details.
  *
  * You should have received a copy of the GNU Affero General Public License
- * along with searx. If not, see < http://www.gnu.org/licenses/ >.
+ * along with SearXNG. If not, see < http://www.gnu.org/licenses/ >.
  *
  * (C) 2019 by Alexandre Flament
  */
```
Comment (return42) [inline searx/static/themes/oscar/src/js/01_init.js]: Lets replace by .. SPDX

     # SPDX-License-Identifier: AGPL-3.0-or-later

Comment (dalf) [reply searx/static/themes/oscar/src/js/01_init.js]: According to 
* https://opensource.google/docs/thirdparty/javascript/#spdx 
* https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/

it should be something like this:
```
/**
 * @license
 * (C) Copyright Contributors to the SearXNG project.
 * (C) Copyright Contributors to the searx project (2014 - 2021).
 * SPDX-License-Identifier: AGPL-3.0-or-later
 */
```

or for some files: 
```
/**
 * @license
 * (C) Copyright Contributors to the SearXNG project.
 * (C) Copyright Contributors to the searx project (2014 - 2021).
 * (C) 2014 by Thomas Pointhuber, <thomas.pointhuber@gmx.at>
 * SPDX-License-Identifier: AGPL-3.0-or-later
 */
```

**Comment by return42** [comment]
On a first test I got an exception .. I assume not all is renamed ....

grafik

**Comment by dalf** [comment]
> On a first test I got an exception .. I assume not all is renamed ....

`searx/static/themes/__common__/js/image_layout.js` except `window.searx` to exist.

Fixed locally with a temporary workaround `window.searx = {}` in `searx/static/themes/oscar/src/js/01_init.js`

[EDIT] see the pushed force update.

File: searx/static/themes/oscar/src/js/01_init.js [hunk 2]
```diff
@@ -32,3 +23,5 @@ window.searx = (function(d) {
         translations: JSON.parse(script.getAttribute('data-translations'))
     };
 })(document);
+
+window.searx = {};
```
Comment (dalf) [inline searx/static/themes/oscar/src/js/01_init.js:27]: Temporary fix for `searx/static/themes/__common__/js/image_layout.js` : 
https://github.com/searxng/searxng/blob/a582cf3d8231f5ed8a881aa87576dfc0600e1c07/searx/static/themes/__common__/js/image_layout.js#L161

File: searx/static/themes/oscar/src/js/01_init.js (hunk 1 above)
Comment (return42) [reply searx/static/themes/oscar/src/js/01_init.js]: >> Lets replace by .. SPDX

> it should be something like this: ...

perfect!

File: searx/static/themes/oscar/src/js/01_init.js (hunk 2 above)
Comment (return42) [reply searx/static/themes/oscar/src/js/01_init.js:27]: > Temporary fix for `searx/static/themes/__common__/js/image_layout.js` ..

Fixes the issue / thanks!  .. could you have an eye on _"temporary"_

**Review by return42 — APPROVED** [review]

File: searx/static/themes/oscar/src/js/01_init.js (hunk 2 above)
Comment (dalf) [reply searx/static/themes/oscar/src/js/01_init.js:27]: `git grep -i "searx[^n]"` ?

**Pull request merged**